test/fstapp_SUITE.erl

```diff
@@ -4,26 +4,35 @@
 
 -compile(export_all).
 
-init_per_suite(Config) ->
-	application:ensure_all_started(fstapp),
+init_per_testcase(_, Config) ->
+	true = register(fstapp_SUITE_process, self()),
+	_ = application:ensure_all_started(fstapp),
 	Config.
 
-end_per_suite(Config) ->
-	application:stop(fstapp),
-	Config.
+end_per_testcase(_, _Config) ->
+	application:stop(fstapp).
 
 all() ->
-	[my_test_case].
+        [external_case, internal_case].
```

Comment:
why those names? what about `frequency_test` and `metrics_test` or something similar?